### DISCUSSION TITLE
Questions about RSUs

### DISCUSSION THREAD

**Discussion by haoting-addepar** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#223] [Q&A]
Please pardon my lack of domain knowledge, but I have two questions about RSU

- My understanding is that there is no strike price for RSUs. However, `exercise_price` is a required field in`PlanSecurityIssuance`, so what's the best way to interpret that when comp type is RSU

- How to interpret the `PlanSecurityRelease` transaction. Is that only apply to RSUs ( because for options there is a separate `PlanSecurityExercise`). Since there is a `release_price` field, can I interpret this as a purchase, i.e., the company is buying back RSUs? or release here means releasing the restricted nature of the RSUs, and I can view this like an exercise event where the RSU is converted into the underlying regular stocks. 

thanks

**Comment by JSv4** [comment]
Hey @haoting-addepar, sorry for the delay. We're wrapping up our outstanding issues for our upcoming public preview and member summit in July. Will try to give this the proper level of thought and get back to you before then, but I'm pretty busy, sorry. If you have thoughts on how this should work (or what makes sense to you), please do feel free to kick this off!
